package.json

```diff
@@ -11,7 +11,7 @@
     "history": "1.17.0",
     "jquery": "^1.11.3",
     "moment": "^2.19.x",
-    "node-sass": "^3.3.3",
+    "node-sass": "^3.13.1",
```

Comment:
Are there any interesting changes we should be aware of here as this seems to be a large jump in versions? The [npm page](https://www.npmjs.com/package/node-sass) suggests we need `node-sass` `4.14+` for compatibility with Node 14 😬

Reply:
Although it does appear that we're quite a way behind the curve here already?